internal/cli/views/basics/form/form.go

```diff
@@ -0,0 +1,212 @@
+package form
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	btn "iwakho/gopherkeep/internal/cli/views/basics/button"
+	"iwakho/gopherkeep/internal/model"
+
+	"github.com/charmbracelet/bubbles/cursor"
+	"github.com/charmbracelet/bubbles/textinput"
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+)
+
+var (
+	focusedStyle        = btn.FocusedStyle
+	cursorStyle         = btn.FocusedStyle
+	noStyle             = lipgloss.NewStyle()
+	helpStyle           = btn.BlurredStyle
+	cursorModeHelpStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("244"))
+)
+
+type Control interface {
+	Submit(p model.Pair) error
+}
+
+type ModelForm struct {
+	Name        string
+	focusIndex  int
+	inputs      []textinput.Model
+	cursorMode  cursor.Mode
+	NextPage    func()
+	buttons     []string
+	indexMax    int
+	failMessage string
+	isUpdated   bool
+	Control
+}
+
+type FormCaller struct {
+	FormName    string
+	InputNames  []string
+	ButtonNames []string
+}
+
+func InitForm(fc FormCaller) ModelForm {
```

Comment:
Я бы советовал использовать указатели в качестве тиаов для передаваемых и возвращаемых значений.
```suggestion
func InitForm(fc *FormCaller) *ModelForm {
```